maintainers/maintainer-list.nix

```diff
@@ -9175,4 +9175,10 @@
     github = "deifactor";
     githubId = 30192992;
   };
+  pulsation = {
```

Comment:
IIRC, adding yourself as a maintainer is preferred to be a separate commit.

Something like, `maintainers: add pulsation`. Then apply the quisk commit.